### PR TITLE
Cosmetic

### PR DESCRIPTION
**Pull request by gilevskaya**

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -1,5 +1,17 @@
 import { LogLevel } from './Logger';
 
+function getLevelShortName(level: string): string {
```
Comment (doboy) [inline src/utils.ts]: I do not like using short names, I think they are not intuitive -- No other logger does this. Maybe we can just remove the log level entirely?

File: src/utils.ts [hunk 2]
```diff
@@ -23,9 +23,9 @@ function formatDate(date: Date) {
   return `${date.getFullYear()}-${month}-${day} ${hours}:${minutes}:${seconds}.${ms}`;
 }
 
-export function logPrefix(logLevel: LogLevel) {
+export function logPrefix(_logLevel?: LogLevel) {
   const time = `${formatDate(new Date())}`;
-  const level = `${LogLevel[logLevel]}`;
+  // const level = `${LogLevel[logLevel]}`;
```
Comment (doboy) [inline src/utils.ts]: remove this if not used

File: src/utils.ts [hunk 3]
```diff
@@ -23,9 +23,9 @@ function formatDate(date: Date) {
   return `${date.getFullYear()}-${month}-${day} ${hours}:${minutes}:${seconds}.${ms}`;
 }
 
-export function logPrefix(logLevel: LogLevel) {
+export function logPrefix(_logLevel?: LogLevel) {
```
Comment (doboy) [inline src/utils.ts:26]: remove _logLevel

File: src/utils.ts (hunk 2 above)
Comment (gilevskaya) [reply src/utils.ts]: OK

File: src/utils.ts (hunk 3 above)
Comment (gilevskaya) [reply src/utils.ts:26]: This is a lot changes in the code, I'll keep it inactive in case we'll decide to bring back the type labels

**Pull request merged**